rpcs3/Emu/RSX/Overlays/overlays.cpp

```diff
@@ -143,7 +143,7 @@ std::u16string ascii8_to_utf16(const std::string& ascii_string)
 
 	for (const auto& code : ascii_string)
 	{
-		out.push_back((char16_t)code);
+		out.push_back(static_cast<char16_t>(code));
```

Comment:
Again unrelated to PR, but you should be able to do exact same pushing directly to `std::u16string` without an intermediate vector?